Kommunicate/Classes/Kommunicate.swift

```diff
@@ -474,7 +473,7 @@ open class Kommunicate: NSObject,Localizable{
 
     }
 
-    private class func  displayAlert(viewController:ALKConversationListViewController) -> UIAlertController {
+    private class func  displayAlert(viewController:KMConversationListViewController) -> UIAlertController {
 
         let alertTitle =  NSLocalizedString("WaitMessage", value: "Please wait...", comment: "")
```

Comment:
Can you replace this with our localized string method? Otherwise, it won't read from a different file.